<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>467</width>
    <height>521</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>10</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>내 차량 조회 </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>80</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>차량번호</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="editCarnum">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>80</y>
      <width>113</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSearch">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>60</y>
      <width>121</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>조회</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>411</width>
      <height>271</height>
     </rect>
    </property>
    <property name="title">
     <string>조회</string>
    </property>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>정기권 :</string>
     </property>
    </widget>
    <widget class="QDateEdit" name="EditStart">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>70</y>
       <width>110</width>
       <height>26</height>
      </rect>
     </property>
    </widget>
    <widget class="QDateEdit" name="EditEnd">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>70</y>
       <width>110</width>
       <height>26</height>
      </rect>
     </property>
    </widget>
    <widget class="QGroupBox" name="Parking">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>381</width>
       <height>71</height>
      </rect>
     </property>
     <property name="title">
      <string>입출차</string>
     </property>
     <widget class="QDateTimeEdit" name="EditIntime">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>30</y>
        <width>261</width>
        <height>26</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>121</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>입차 시간:</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="OUT">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>190</y>
       <width>381</width>
       <height>61</height>
      </rect>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QDateTimeEdit" name="EditOuttime">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>20</y>
        <width>261</width>
        <height>26</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>121</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>출차 시간:</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="btnConfirm">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>440</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>확인</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
